packages/capabilities/src/types.ts

```diff
@@ -271,13 +283,21 @@ export interface StoreListItem {
   link: UnknownLink
   size: number
   origin?: UnknownLink
+  insertedAt: string
```

Comment:
add insertedAt to StoreListItem as we already want to return this info for list usage, and it means we can re-use the type for both list and get operations.